specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -1220,14 +1226,15 @@
           "DeletedSecrets"
         ],
         "operationId": "PurgeDeletedSecret",
-        "description": "Permanently deletes the specified secret. aka purges the secret. Authorization: requires the secrets/purge permission.",
+        "summary": "Permanently deletes the specified secret.",
+        "description": "The purge deleted secret operation removes the secret permanently, without the possibility of recovery. It requires the secrets/purge permission, which must be assigned explicitly.",
```

Comment:
>The purge deleted secret operation removes the secret permanently, without the possibility of recovery. It requires the secrets/purge permission, which must be assigned explicitly. [](start = 24, length = 180)

should this mention that it can only be performed on a soft delete enabled vault?